sources/models/shared-defaults/defaults.toml

```diff
@@ -162,9 +162,12 @@ affected-services = ["bootstrap-containers"]
 # Certdog
 
 [services.pki]
-configuration-files = []
+configuration-files = ["certdog-toml"]
 restart-commands = ["/usr/bin/certdog"]
 
+[metadata.settings.pki]
+affected-services = ["pki"]
```

Comment:
This is already defined in:
* `containerd-cri-pki.toml`
* `docker-pki.toml`

So you'll need to update both of those instead, and conditionalize the migration.

Reply:
Should i just remove this from defaults.toml and leave the migration to simply the add of the tempalte file?

Reply:
> Should i just remove this from defaults.toml and leave the migration to simply the add of the tempalte file?

No - you need to update `defaults.toml` so that freshly launched instances with an unpopulated datastore get the change. And you need migrations so that instances that upgrade or downgrade get the new values on upgrade, and the old values on downgrade.

Reply:
Oh, you're right - you can drop this since `certdog` is already invoked as the restart command, and the affected services aren't changing.